packages/terra-navigation-layout/src/terra-dev-site/doc/example/HeaderExample.scss

```diff
@@ -0,0 +1,16 @@
+@import './NavigationLayoutDocCommon';
```

Comment:
Lets add all the styles directly that are needed, or we can import `NavigationLayoutDocCommon` directly into the examples and bind those class names

Reply:
Fixed here https://github.com/cerner/terra-framework/pull/742/commits/d320272a142e98bfee8b1d612816f9fabd7aa36d.